maro/rl/policy/dqn.py

```diff
@@ -325,6 +341,35 @@ def _update_target(self):
         self.target_q_net.soft_update(self.q_net, self.soft_update_coeff)
         self._target_q_net_version = self._q_net_version
 
+    def distributed_learn(self, rollout_info, host_id_list, name, proxy):
```

Comment:
The use of "host" here is confusing. A "host" is meant to house a policy instance and run its training loop.  The grad computation should be done by a set of stateless workers named "grad worker" or something like that.